static/app/components/feedback/feedbackItem/feedbackScreenshot.tsx

```diff
@@ -42,6 +49,20 @@ export default function FeedbackScreenshot({
       )}
       {onClick ? <StyledImageButton onClick={onClick}>{img}</StyledImageButton> : img}
     </StyledPanel>
+  ) : (
+    <File onClick={onClick}>
+      <NoPreviewFound>
+        <IconImage />
+        {t('No preview found')}
+      </NoPreviewFound>
+      <Tooltip position="right" title={t('Click to download')}>
+        <FileDownload
+          href={`/api/0/projects/${organization.slug}/${projectSlug}/events/${screenshot.event_id}/attachments/${screenshot.id}/?download=1`}
+        >
```

Comment:
dogfooding, but i wonder if we should set an extra attr so this doesn't get detected as a dead click

Reply:
added a `className` so we can add it to our `slowClickIgnoreSelectors` if we want